docs/user/setup.rst

```diff
@@ -3,49 +3,65 @@
 Environment setup
 =================
 
-This is what you will be doing, in a nutshell:
+Scala Native has the following system requirements:
 
-* installation of sbt
-* installation of LLVM and Clang
-* installation of Boehm GC
+* Java 8 or later
+* sbt 0.13.15
```

Comment:
Yes, that's true. What do you think about removing the mention of `project/build.properties` from the [sbt chapter](http://www.scala-native.org/en/latest/user/sbt.html)? If we specify it in the setup chapter, it would be redundant.